### ISSUE TITLE
Incorporate SwiftUI support

### ISSUE THREAD

**Issue by mattmassicotte** [ChimeHQ/Textbook#1]
I'm finally going to give this repo some attention. @unnamedd would you like to start looking at ways to incorporate your SwiftUI work?

**Comment by unnamedd** [comment]
> I'm finally going to give this repo some attention. @unnamedd would you like to start looking at ways to incorporate your SwiftUI work?

Definitely! I do apologize for being so far lately, the time here isn't being so friendly to me, I will get back to Discord though to finally start this joint work!

**Comment by mattmassicotte** [comment]
First: you are not allowed to apologize for things like this!
Second: Don't be shy about using GitHub for discussion too. I'll make sure to echo important details from discord here too.
